x/feegrant/basic_fee_test.go

```diff
@@ -1,16 +1,20 @@
 package feegrant_test
 
 import (
+	"context"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"cosmossdk.io/core/header"
+	"cosmossdk.io/log"
 	storetypes "cosmossdk.io/store/types"
 	"cosmossdk.io/x/feegrant"
 
+	corecontext "cosmossdk.io/core/context"
+	"github.com/cosmos/cosmos-sdk/runtime"
```

Comment:
we are trying to avoid `runtime/v2` imports in modules. IMHO we should do the same for runtime v0

Reply:
got it, makes sense to avoid dependency import